pkg/cnb/cnb_metadata.go

```diff
@@ -55,8 +66,19 @@ func (r *RemoteMetadataRetriever) GetBuilderImage(repo registry.ImageRef) (Build
 		return BuilderImage{}, errors.Wrap(err, "failed to retrieve builder image SHA")
 	}
 
+	runImage, err := r.RemoteImageFactory.NewRemote(registry.NewNoAuthImageRef(metadata.Stack.RunImage.Image))
```

Comment:
Should this be NoAuthImageRef?

Reply:
This is WIP where we are trying to get it to work first with public run images..will have to rethink how this works for private run images..we might have to make the runimage a type that can be an image ref that rides on things from the builder..